components/tabstrip/tabs-collection.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Tabs Collection
+page_title: Tabs Collection
+description: Overview of the TabStrip for Blazor.
+slug: tabstrip-tabs-collection
+tags: telerik,blazor,tab,strip,tabstrip,collection
+published: True
+position: 17
+---
+
+# Tabs Collection
```

Comment:
Shouldn't we use the component name in all H1 headings (and possibly - page_title) ?

Reply:
Updated.